src/tribler-gui/tribler_gui/widgets/channelcontentswidget.py

```diff
@@ -103,7 +104,11 @@ def on_channel_committed(self, response):
         if response and response.get("success", False):
             if not self.autocommit_enabled:
                 self.commit_control_bar.setHidden(True)
-            if self.model:
+            if (
+                self.model
+                and self.model.channel_info.get("state", None) == "Personal"
+                and self.model.channel_info.get("dirty", False)
+            ):
```

Comment:
**nit:** the `on_channel_committed` function may be simplified:

```python
   def on_channel_committed(self, response):
        if not response or not response.get("success", False):
            return

        if not self.autocommit_enabled:
            self.commit_control_bar.setHidden(True)

        if not self.model:
            return

        info = self.model.channel_info
        if info.get("state") == "Personal" and info.get("dirty", False):
            self.model.reset()
            self.update_labels()
```

Reply:
Thnx!